LinkedListByEnum/LinkedListByEnum/ViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  ViewController.swift
+//  LinkedListByEnum
+//
+//  Created by Maksym Humeniuk on 4/18/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        var linkedList = LinkedList<Int>()
+        linkedList.append(value: 10)
+        linkedList.append(value: 20)
+        linkedList.append(value: 30)
+        linkedList.append(value: 40)
+        linkedList.append(value: 50)
+        
+        print(linkedList);
+        
+        do{
+            let i = try linkedList.getItem(item: 0)
+            print(i)
+        } catch {
+            print("Something wrong")
+        }
+
+    }
+    
+    
+}
+
+
+indirect enum LinkedList <T> : CustomStringConvertible{
+
+    case Empty
+    case Node(value: T, next: LinkedList)
+    
+    init(){self = .Empty}
+
+    mutating func append(value: T){
+        self = addValueToLastLevel(value: value, list: self)
```

Comment:
this is adding new values into the had of the list but append should add to the end